app/controllers/listings.controller.js

```diff
@@ -13,4 +13,41 @@ exports.list = function (req, res, next) {
             }
         }).populate('user');
     }
-};
\ No newline at end of file
+};
+
+exports.createListing = function(req, res, next) {
+    var name = (req.body.name) ? req.body.name.trim() : '';
+    var price = req.body.price;
```

Comment:
It might be a good idea to go ahead and parse the price here using `parseFloat()` and limit it to two decimal places using `toFixed(2)`.